datamonster_api/tests/lib/test_datamonster.py

```diff
@@ -141,18 +181,18 @@ def test_get_data_1(
         "time_span",
         "value",
     ]
-    # size sanity check
     assert len(df) == 8
 
-    # Check the first row
-    assert df.iloc[0]["dimensions"] == {"category": "Whole Foods"}
+    assert df.iloc[0]["dimensions"] == {"category": "Whole Foods", "country": "US"}
```

Comment:
check why this changed

Reply:
ah - because in conftest we have `"splitColumns": ["category"]` but are now going off the actual schema